pandas/io/parsers.py

```diff
@@ -40,70 +48,79 @@ class ParserWarning(Warning):
     use the first column as the index (row names)
-names : array-like, default None
-    List of column names to use. If file contains no header row, then you
-    should explicitly pass header=None
-prefix : string, default None
-    Prefix to add to column numbers when no header, e.g 'X' for X0, X1, ...
-na_values : str, list-like or dict, default None
-    Additional strings to recognize as NA/NaN. If dict passed, specific
-    per-column NA values
+usecols : array-like, default None
+    Return a subset of the columns.
+    Results in much faster parsing time and lower memory usage.
+squeeze : boolean, default False
+    If the parsed data only contains one column then return a Series
+prefix : str, default None
+    Prefix to add to column numbers when no header, e.g. 'X' for X0, X1, ...
+mangle_dupe_cols : boolean, default True
+    Duplicate columns will be specified as 'X.0'...'X.N', rather than 'X'...'X'
+dtype : Type name or dict of column -> type, default None
+    Data type for data or columns. E.g. {'a': np.float64, 'b': np.int32}
+    (Unsupported with engine='python'). Use `str` or `object` to preserve and
+    not interpret dtype.
+%s
+converters : dict, default None
+    Dict of functions for converting values in certain columns. Keys can either
+    be integers or column labels
 true_values : list, default None
     Values to consider as True
 false_values : list, default None
     Values to consider as False
+skipinitialspace : boolean, default False
+    Skip spaces after delimiter.
+skiprows : list-like or integer, default None
+    Line numbers to skip (0-indexed) or number of lines to skip (int)
+    at the start of the file
+skipfooter : int, default 0
+    Number of lines at bottom of file to skip (Unsupported with engine='c')
+nrows : int, default None
+    Number of rows of file to read. Useful for reading pieces of large files
+na_values : str or list-like or dict, default None
+    Additional strings to recognize as NA/NaN. If dict passed, specific
+    per-column NA values.  By default the following values are interpreted as
+    NaN: `'""" + "'`, `'".join(sorted(_NA_VALUES)) + """'`.
 keep_default_na : bool, default True
     If na_values are specified and keep_default_na is False the default NaN
-    values are overridden, otherwise they're appended to
-parse_dates : various, default False
-
+    values are overridden, otherwise they're appended to.
+na_filter : boolean, default True
+    Detect missing value markers (empty strings and the value of na_values). In
+    data without any NAs, passing na_filter=False can improve the performance
+    of reading a large file
+verbose : boolean, default False
+    Indicate number of NA values placed in non-numeric columns
+skip_blank_lines : boolean, default True
+    If True, skip over blank lines rather than interpreting as NaN values
+parse_dates : boolean or list of ints or names or list of lists or dict, \
```

Comment:
This had been changed to "various" to obey the line character limit, I replaced with '\' but I don't have strong feelings either way.